webapp.yml

```diff
@@ -0,0 +1,83 @@
+# Description
+# ===========
+# This playbook creates an Azure Linux Web app with Java container Tomcat 8.5,
+# then build a Java hello world app, ftp the war file to the Web app.
+# Please note, it's recommended to use CI/CD tool to do webapp deployment, such as Jenkins.
+# Deployment step in this sample is just to validate webapp created is working.
```

Comment:
…to validate that web app is created

Reply:
deployment step is not to validate webapp is created, but to validate the webapp is working as configured: java webapp deployed to  it works.

Reply:
so the wording should be changed a bit, like "to validate that created webapp is working." or something like this